soh/soh/Enhancements/randomizer/3drando/location_access/locacc_water_temple.cpp

```diff
@@ -122,7 +122,7 @@ void AreaTable_Init_WaterTemple() {
 
   areaTable[WATER_TEMPLE_SOUTH_LOWER] = Area("Water Temple South Lower", "Water Temple", WATER_TEMPLE, NO_DAY_NIGHT_CYCLE, {}, {
                   //Locations
-                  LocationAccess(WATER_TEMPLE_GS_BEHIND_GATE, {[]{return CanUse(HOOKSHOT) || (IsAdult && CanUse(HOVER_BOOTS));}}),
+                  LocationAccess(WATER_TEMPLE_GS_BEHIND_GATE, {[]{return CanUse(HOOKSHOT) || (IsAdult && (CanJumpslash || CanUseProjectile) && CanUse(HOVER_BOOTS));}}),
```

Comment:
```suggestion
                  LocationAccess(WATER_TEMPLE_GS_BEHIND_GATE, {[]{return (CanJumpslash || CanUse(MEGATON_HAMMER)) && (CanUse(HOOKSHOT) || (IsAdult && CanUse(HOVER_BOOTS)));}}),
```
you cant get past the gate without a jumpslash item as far as i can tell (even explosives dont reach)